storage/pkg/flagset/users.go

```diff
@@ -20,55 +20,48 @@ func UsersWithConfig(cfg *config.Config) []cli.Flag {
 
 		// Services
 
-		// Users
+		// Userprovider
```

Comment:
I renamed the STORAGE_USERS prefix for the userprovider service to STORAGE_USERPROVIDER, this way I could use STORAGE_USERS for the storage provider mounted at `/users`